heal-commands.go

```diff
@@ -296,6 +296,17 @@ func (adm *AdminClient) Heal(ctx context.Context, bucket, prefix string,
 	return healStart, healTaskStatus, nil
 }
 
+// MRFStatus exposes MRF metrics of a server
+type MRFStatus struct {
+	BytesHealed uint64 `json:"bytesHealed"`
+	ItemsHealed uint64 `json:"itemsHealed"`
```

Comment:
```suggestion
	ObjectsHealed uint64 `json:"itemsHealed"`
```

Reply:
Also let's follow either `key_value` style or `keyValue` style - let's not mix both for JSON tags.